cpp/ql/test/library-tests/ir/ir/PrintAST.expected

```diff
@@ -67,7 +67,7 @@ bad_asts.cpp:
 #    5|   params: 
 #-----|     0: [Parameter] p#0
 #-----|         Type = [RValueReferenceType] S &&
-#    9| [MemberFunction] int Bad::S::MemberFunction(int)
+#    9| [FunctionTemplateInstantiation,MemberFunction] int Bad::S::MemberFunction<int>(int)
```

Comment:
I'd expect to see `<6>` here instead of `<int>`. I suppose the function signature pretty-printer needs to learn about `getTemplateArgumentValue` to find the `6`, but where is the `int` coming from?

Reply:
I'm still working on getting this tidied-up

Reply:
If it causes you trouble, feel free to throw it over the fence to the analysis team.